concepticondata/concepticon.tsv

```diff
@@ -3779,3 +3779,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3793	EIGHT THOUSAND	Quantity	The natural number eight thousand (8000).	Number	
 3794	TWO MILLION	Quantity	The natural number two million (2000000).	Number	
 3795	DARK GREEN	Sense perception	A darker variant of the color green, mixed with black, with an approximate wavelength between 543 and 549 nm.	Property	
+3796	ALBATROSS	Animals	Any seabird of the family Diomedeidae. Some species are among the largest flying birds in the world.	Person/Thing	
+3797	OSTRICH	Animals	Any of two species of large, flightless birds in the family Struthionidae, native to Africa. 	Person/Thing	
```

Comment:
I'd add "of" in "in the family of Struthionidae".

Reply:
no, that sounds weird :) -- biologists say "family X"

Reply:
Oh ok, good to know :) We have a few definitions where "family of" is used, so I had this wording in mind.

Reply:
Ahh, then consistency is more important, so go with what everything else is.

Reply:
I just quickly checked some descriptions of birds that first came to mind - parrot, eagle, sparrow, hummingbird - and there, it is always "family" without "of". So, if that sounds more natural and both versions exist in the definitions, I'm going to leave it as it is. Thanks for reviewing!